backend/app/api/api_v1/routers/search.py

```diff
@@ -31,4 +31,5 @@ def search_documents(
     return _OPENSEARCH_CONNECTION.query(
         search_request_body=search_body,
         opensearch_internal_config=_OPENSEARCH_INDEX_CONFIG,
-    )
+        preference=str(current_user.id)
+    )
```

Comment:
this is weird, python files should end in a newline character... I would have thought flake8 would complain about that. no need to fix it, it'll be fixed automatically later.